app/models/ai_persona.rb

```diff
@@ -56,6 +60,23 @@ def self.all_personas
       .map(&:class_instance)
   end
 
+  def self.mentionables
+    persona_cache[:mentionable_usernames] ||= AiPersona
+      .where(mentionable: true)
+      .where(enabled: true)
+      .joins(:user)
+      .pluck("ai_personas.id, users.id, lower(users.username), allowed_group_ids, default_llm")
```

Comment:
We already store `username_lower` on users:

```suggestion
      .pluck("ai_personas.id, users.id, users.username_lower, allowed_group_ids, default_llm")
```